Tests/KituraTests/TestRequests.swift

```diff
@@ -123,4 +124,40 @@ class TestRequests: XCTestCase {
 
         return router
     }
+
+    func testParameters() {
```

Comment:
Can we expand the tests a little more? I feel like this is just the basic "hello world" test and there must be more complicated examples we should cover as well.